PlayerUI/Views/PUIPlayerView.swift

```diff
@@ -740,22 +737,21 @@ public final class PUIPlayerView: NSView {
         controlsContainerView.layer?.masksToBounds = false
         controlsContainerView.layer?.zPosition = 10
 
+        scrimContainerView = PUIScrimView(frame: controlsContainerView.bounds)
+
         addSubview(scrimContainerView)
         addSubview(controlsContainerView)
 
         scrimContainerView.translatesAutoresizingMaskIntoConstraints = false
         scrimContainerView.leadingAnchor.constraint(equalTo: leadingAnchor).isActive = true
         scrimContainerView.trailingAnchor.constraint(equalTo: trailingAnchor).isActive = true
-        scrimContainerView.topAnchor.constraint(equalTo: topAnchor).isActive = true
         scrimContainerView.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
+        scrimContainerView.heightAnchor.constraint(equalTo: controlsContainerView.heightAnchor, multiplier: 1.4, constant: 0).isActive = true
```

Comment:
Not super happy with use of the multiplier here -- I'm not sure if I'm anchoring to the wrong thing, but the height wasn't behaving as expected at all 🤔